src/coreclr/tests/src/JIT/Regression/CLR-x86-JIT/V2.0-Beta2/b426654/b426654.il

```diff
@@ -30,7 +30,7 @@
   .field static assembly int32 s_Zero
   .field static assembly int32 s_i
   .field private static class cs1 s_obj
-  .field private static bool s_timeUp
```

Comment:
`modreq([mscorlib]System.Runtime.CompilerServices.IsVolatile)` is unnecessary cruft. The only thing that matters for the runtime is the `volatile.` prefix for the IL instructions.

Reply:
Thanks for pointing that out, removed in 2nd commit.